### DISCUSSION TITLE
Justify Text

### DISCUSSION THREAD

**Discussion by igomez840922** [QuestPDF/QuestPDF#761] [General]
Hi, when you are going to implement the text justify align, we are using quest for a long and that's the only issue we are dealing with our customers from the beginning.
Any idea when this will be done ?

**Comment by girlpunk** [comment]
https://github.com/QuestPDF/QuestPDF/issues/535#issuecomment-1894534389

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. We apologize for the delayed response. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

Text justification is now fully supported via the [Justify](https://www.questpdf.com/api-reference/text/paragraph-style.html) method — simply call `.Text("...").Justify()` on your content.

Should you encounter any further problems, please don't hesitate to open a new issue.
Thank you!
